### PR TITLE
SortBy Class

### PR DESCRIPTION
**Pull request by mracos**
References #6 

parse nodes
-----------------

- [x] TypeName
- [x] TypeCast
- [ ] FuncCall
- [ ] A_Indices
- [ ] A_Indirection
- [ ] A_ArrayExpr
- [ ] MultiAssignRef
- [X] SortBy
- [ ] SortGroupClause
- [ ] GroupingSet

priming nodes
-------------------
- [ ] IntoClause
- [ ] Var
- [ ] CaseExpr
- [ ] ArrayExpr
- [ ] NullTest
- [ ] BooleanTest
- [ ] FromExpr
- [ ] OpExpr

### CODE REVIEW

File: psqlparse/nodes/parsenodes.py [hunk 1]
```diff
@@ -251,3 +251,15 @@ def __init__(self, obj):
 
     def tables(self):
         return set()
+
+class SortBy(Node):
```
Comment (alculquicondor) [inline psqlparse/nodes/parsenodes.py]: You need to import the class in `psqlparse/nodes/__init__.py`

File: test/test_parse.py [hunk 2]
```diff
@@ -128,6 +128,16 @@ def test_select_case(self):
...
         self.assertEqual(len(stmt.target_list), 2)
+    
+    def test_select_order_by(self):
+        query = "SELECT * FROM my_table ORDER BY my_table.id"
+        stmt = parse(query).pop()
+
+        self.assertIsInstance(stmt, nodes.SelectStmt)
+
+        orderby = stmt.sort_clause.pop()
+        self.assertIsInstance(orderby, nodes.parsenodes.SortBy)
```
Comment (alculquicondor) [inline test/test_parse.py]: please include assertions for some properties

File: psqlparse/nodes/parsenodes.py [hunk 3]
```diff
@@ -251,3 +251,15 @@ def __init__(self, obj):
 
     def tables(self):
         return set()
+
```
Comment (alculquicondor) [inline psqlparse/nodes/parsenodes.py]: add an extra space here please (PEP8)

**Comment by ffmarcos** [comment]
I don't know if here is the place, but what you think about running style test (flake8) on travis?

File: test/test_parse.py (hunk 2 above)
Comment (alculquicondor) [inline test/test_parse.py]: I know :)

**Comment by alculquicondor** [comment]
+1 on leveraging travis for doing that

File: test/test_parse.py [hunk 4]
```diff
@@ -129,14 +129,19 @@ def test_select_case(self):
...
-
         orderby = stmt.sort_clause.pop()
+
         self.assertIsInstance(orderby, nodes.parsenodes.SortBy)
+        self.assertEqual(len(orderby.node.fields), 2)
+
+        name_column = orderby.node.fields.pop()
+        name_table = orderby.node.fields.pop()
+
+        self.assertEqual(name_column.__str__(), 'id')
```
Comment (alculquicondor) [inline test/test_parse.py]: use `str(name_column)` instead

**Comment by alculquicondor** [comment]
@ffmarcos maybe you should finish with the implemented classes (finishing up the tests) and let's merge.
Then you open others PR for other sets of classes :)

**Pull request closed without merging**